thorlcr/graph/thgraphslave.cpp

```diff
@@ -1775,14 +1775,34 @@ IReplicatedFile *createEnsurePrimaryPartFile(CActivityBase &activity, const char
 class CFileCache;
 class CLazyFileIO : public CInterface, implements IFileIO, implements IDelayedFile
 {
+    class CFileIOWrapper : public CSimpleInterfaceOf<IFileIO>
```

Comment:
Should be removed, I will do so.
Though there's a slight caveat/oddity involved here that I'm not sure how to address.
The remote key manager gets the underlying 'real' IFileIO from the lazy io via getRealFileIO().
Which means it can live slightly longer and outside the lazy fileIO cache.

The lazyfileio cache updates the read stats on removal (on IFileIO::close).
Since the remote io doesn't involve any direct IFileIO calls, that's okay as far as these stats are concerned.
Local key stats would be kept via jhtree 
Remote key stats would be tracked remotely, but should probably be captured here too somehow.